measure-runtime zsh macOS often exceed 275 ms on CI

```
min in 10 runs was: 290.472 ms
1744
Expected was: 275 ms
1745
Error: Process completed with exit code 1.
```
for example [this run](https://github.com/kaihowl/dotfiles/runs/4593881370?check_suite_focus=true).

This seems to happen more often now. Time to check the startup of zsh again.

--- a/zsh/zshrc.symlink
+++ b/zsh/zshrc.symlink
@@ -24,9 +24,6 @@ fi
 # Periodically (typically once a day), update from github
 source $DOTS/script/check_for_upgrade.sh
 
-# First, load oh-my-zsh
-source $DOTS/zsh/oh-my.zsh.preamble
-
 # your project folder that we can `c [tab]` to
 export PROJECTS=~/Documents/repos/
 
@@ -54,13 +51,12 @@ do
   source $file
 done
 
-# initialize autocomplete here, otherwise functions won't be loaded
-autoload -U compinit
-compinit
-
 unset config_files
 
 [ -f ~/.fzf.zsh ] && source ~/.fzf.zsh
+
+# Last, load oh-my-zsh (which does compinit)
+source $DOTS/zsh/oh-my.zsh.preamble
 
 # We don't want this and oh-my-zsh has no way to turn it off
 # Hence, we brute-force this back

